CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 **Unreleased**
 
 - Update BedLikeTrack to display strand-specific entries
+- Update dependencies to support Node.js 12 and 13
```

Comment:
Not sure we track technical dept things unless it's a breaking or major change. @pkerpedjiev Any thoughts?

Reply:
Yeah, I suppose that's not necessary. I created a PR to update our CONTRIBUTING.md file with this information: https://github.com/higlass/higlass/pull/810